### DISCUSSION TITLE
[Feature] Link device image album to immich album

### DISCUSSION THREAD

**Discussion by bitfl0wer** [immich-app/immich#4266] [Feature Request]
### The feature

For me, iOS handles albums quite well. I like that it automatically sorts Screenshots into a Screenshots folder, Live Photos into a Live Photo album, etc. 

I'd love to be able to "link" local albums to immich created albums, so that I could preserve this order within my immich instance.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678
